### PR TITLE
Patch diagnostics

### PR DESCRIPTION
**Pull request by arnaudlh**

### CODE REVIEW

File: examples/compute/virtual_machine/211-vm-bastion-winrm-agents/standalone/vm_extensions.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 module "vm_extension_monitoring_agent" {
   source     = "../../../../../modules/compute/virtual_machine_extensions"
-  depends_on = [module.caf]
+  # depends_on = [module.caf]
```
Comment (arnaudlh) [inline examples/compute/virtual_machine/211-vm-bastion-winrm-agents/standalone/vm_extensions.tf:7]: Cleanup?

File: examples/compute/virtual_machine/211-vm-bastion-winrm-agents/standalone/vm_extensions.tf [hunk 2]
```diff
@@ -22,7 +22,7 @@ module "vm_extension_monitoring_agent" {
 
 module "vm_extension_diagnostics" {
   source     = "../../../../../modules/compute/virtual_machine_extensions"
-  depends_on = [module.caf]
+  # depends_on = [module.caf]
```
Comment (arnaudlh) [inline examples/compute/virtual_machine/211-vm-bastion-winrm-agents/standalone/vm_extensions.tf:25]: Cleanup?

File: examples/dynamic_secrets.tf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+module dynamic_keyvault_secrets {
+  source  = "aztfmod/caf/azurerm//modules/security/dynamic_keyvault_secrets"
```
Comment (arnaudlh) [inline examples/dynamic_secrets.tf:3]: self calling?

File: modules/recovery_vault/recovery_vault.tf [hunk 4]
```diff
@@ -9,43 +9,97 @@ resource "azurecaf_name" "asr_rg_vault" {
   use_slug      = var.global_settings.use_slug
 }
 
-resource "azurerm_recovery_services_vault" "asr_rg_vault" {
-  name                = azurecaf_name.asr_rg_vault.result
-  location            = var.location
-  resource_group_name = var.resource_group_name
-  sku                 = "Standard"
-  tags                = local.tags
+# resource "azurerm_recovery_services_vault" "asr_rg_vault" {
```
Comment (arnaudlh) [inline modules/recovery_vault/recovery_vault.tf:12]: cleanup?

File: modules/recovery_vault/replication_policy.tf [hunk 5]
```diff
@@ -1,11 +1,12 @@
 
 resource "azurerm_site_recovery_replication_policy" "policy" {
-  depends_on = [azurerm_recovery_services_vault.asr_rg_vault, time_sleep.delay_create]
+  depends_on = [azurerm_resource_group_template_deployment.asr]
+  # depends_on = [azurerm_recovery_services_vault.asr_rg_vault, time_sleep.delay_create]
```
Comment (arnaudlh) [inline modules/recovery_vault/replication_policy.tf:4]: cleanup?

**Review by arnaudlh — COMMENTED** [review]
Mainly cleanup...

File: examples/dynamic_secrets.tf (hunk 3 above)
Comment (LaurentLesle) [reply examples/dynamic_secrets.tf:3]: calling private module from registry

**Review by LaurentLesle — APPROVED** [review]

**Pull request merged**